### ISSUE TITLE
K&R syntax is obsolete, use ANSI

### ISSUE THREAD

**Issue by jeffdaily** [GlobalArrays/ga#16]
There are a few places, e.g. global/src/DP.c, where K&R style function declarations are still in use.  

For example:

```C
// K&R syntax
int foo(a, p) 
    int a; 
    char *p; 
{ 
    return 0; 
}

// ANSI syntax
int foo(int a, char *p) 
{ 
    return 0; 
}
```

Clean these up.  Consider using gcc warnings to locate them, but it might not be bullet proof.
`-Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes` or `-ansi -pedantic`.

**Comment by ajaypanyala** [comment]
Fixed via #172

**Comment by jeffhammond** [comment]
There's no way we fixed K&R in MA but that's okay. The K&R in MA is actually good 😆
